.gitignore

```diff
@@ -213,7 +213,8 @@ $RECYCLE.BIN/
 .LSOverride
 
 # Icon must end with two \r
-Icon


```

Comment:
Apparently this is a vim thing? 

https://stackoverflow.com/questions/5843495/what-does-the-m-character-mean-in-vim

Reply:
This line should not be changed. It's useless otherwise. You can fix this via `git checkout main .gitignore` then `git add -p` and `n` to skip the line.